demo/vl-map-layer-switcher.html

```diff
@@ -24,24 +24,75 @@
+                        <vl-map-layer data-vl-name="Kaartlaag 1" data-vl-features='
+                            {
+                                "type": "FeatureCollection",
+                                "features": [{
+                                    "type": "Feature",
+                                    "geometry": {
+                                        "type": "Point",
+                                        "coordinates": [149055.0,199908.0]
+                                    }
+                                }]
+                            }'>
+                            <vl-map-layer-circle-style data-vl-color="black"></vl-map-layer-circle-style>
+                        </vl-map-layer>
+                        <vl-map-layer data-vl-name="Kaartlaag 2" data-vl-features='
+                            {
+                                "type": "FeatureCollection",
+                                "features": [{
+                                    "type": "Feature",
+                                    "geometry": {
+                                        "type": "Point",
+                                        "coordinates": [154055.0,199908.0]
+                                    }
+                                }]
+                            }'>
+                            <vl-map-layer-circle-style data-vl-color="yellow"></vl-map-layer-circle-style>
+                        </vl-map-layer>
+                        <vl-map-layer data-vl-name="Kaartlaag 3" data-vl-features='
+                            {
+                                "type": "FeatureCollection",
+                                "features": [{
+                                    "type": "Feature",
+                                    "geometry": {
+                                        "type": "Point",
+                                        "coordinates": [159055.0,199908.0]
+                                    }
+                                }]
+                            }'>
+                            <vl-map-layer-circle-style data-vl-color="red"></vl-map-layer-circle-style>
+                        </vl-map-layer>
+                    </vl-map>
                 </div>
+            </div>
+        </vl-demo>
+
+        <vl-demo data-vl-title="Kaart met gepersonaliseerde optie om kaartlagen te tonen en verbergen">
+            <div is="vl-grid" data-vl-is-stacked>
                 <div is="vl-column" data-vl-size="12" data-vl-medium-size="12">
-                    <vl-map id="map-with-layer-switcher">
+                    <vl-map id="map-with-custom-layer-switcher">
                         <vl-map-side-sheet>
                             <vl-map-layer-switcher>
-                                <vl-checkbox data-vl-label="Kaartlaag" data-vl-layer="layer"></vl-checkbox>
+                                <vl-checkbox data-vl-label="Kaartlaag" data-vl-layer="layer-1"></vl-checkbox>
```

Comment:
k weet niet of je evt nog wil "vl-layer-switcher-checkbox" wil introduceren ...
voorlopig (van ons uit) geen vraag naar, maar misschien wel als je hier custom stuff wil doen met tussentitelkes, maar dan toch nog de "checkbox voor 1 laag" wil hergebruiken of zo ...

Reply:
Gezien alleen het `data-vl-layer` attribuut extra is t.o.v. de `vl-checkbox` zie ik nog niet echt een voordeel om deze op dit moment te introduceren. Geef zeker aan wanneer bij toekomstige updates het voor jullie wel interessant lijkt te zijn, op vraag kan dat dan zeker voorzien worden moest het spontaan niet gebeurd zijn.

Reply:
voorlopig zouden wij alleszins beginnen met de "default" versie voor zover ik weet, dus fair enough voor ons